http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/DevModeGrpcIntegrationReactiveIT.java

```diff
@@ -79,6 +85,14 @@ public void testGrpcDevUIServicesView() {
             for (String text : GRPC_SERVICE_VIEW_EXPECTED_CONTENT) {
                 assertTrue(grpcSvcView.contains(text), "DevUI gRPC services view is incomplete: " + grpcSvcView);
             }
+            // search for gRPC service implementation classes differently as they are in a shadow root and sometimes
+            // (like on Windows) they cannot be accessed
+            for (String implClass : GRPC_SERVICE_IMPLEMENTATION_CLASSES) {
+                var locator = page.getByText(implClass);
+                assertNotNull(locator);
+                assertNotNull(locator.textContent());
+                assertTrue(locator.textContent().contains(implClass));
```

Comment:
I have added proposed message everywhere. Truth be told, I think what happens when the text is missing is that `page.getByText` timeouts and Awaitility fails the test. As least that is what happened when I tried to lookup "nonsense" just so that I know the test is not succeeded for every text :-).

Let's have the message there as it makes sense to have it next by assertion.

Reply:
> Truth be told, I think what happens when the text is missing is that page.getByText timeouts and Awaitility fails the test.

It make sense, it wasn't came to my mind when reviewing that.